Yubico.YubiKey/src/Yubico.YubiKey.csproj

```diff
@@ -168,27 +155,25 @@ limitations under the License. -->
 
   </ItemGroup>
 
-  <!-- Clean up DocFX output -->
-  <Target Name="CleanDocFX" AfterTargets="DocClean" Condition="$(Configuration.Contains('WithDocs')) and $(TargetFramework)=='netstandard2.0'">
-    <Message Text="Cleaning up DocFX generated documentation." Importance="high" />
-    <RemoveDir Directories="$(ProjectDir)..\docs\_site\;$(ProjectDir)..\docs\memberpage\" />
-    <ItemGroup Label="DocFX Artifacts to Clean">
-      <FilesToDelete Include="$(ProjectDir)..\docs\core-api\*.yml" />
-      <FilesToDelete Include="$(ProjectDir)..\docs\core-api\.manifest" />
-      <FilesToDelete Include="$(ProjectDir)..\docs\yubikey-api\*.yml" />
-      <FilesToDelete Include="$(ProjectDir)..\docs\yubikey-api\.manifest" />
-      <FilesToDelete Include="$(ProjectDir)..\docs\log.txt" />
-    </ItemGroup>
-    <Delete Files="@(FilesToDelete)" />
+  <!-- Build documentation -->
+  <!-- DocFX is a tool that generates documentation from source code. It is used to generate the API documentation for the YubiKey SDK. -->
+  <!-- Run this before submitting a PR to make sure that your code does not produce any documentation issues -->
+  <!-- This can be called either by command line or in VS Code Tasks (.vscode/tasks.json) 
+    > dotnet msbuild Yubico.YubiKey/src/Yubico.YubiKey.csproj /t:DocFXBuild 
+  -->
+  <Target Name="DocFXBuild" DependsOnTargets="CheckDocFX;Restore;Build">
+    <Exec Command="docfx build ../../docfx.json --logLevel warning --log ../docs/docfx.log --warningsAsErrors" IgnoreExitCode="true">
+      <Output TaskParameter="ExitCode" PropertyName="DocFXExitCode" />
+    </Exec>
+    <!-- Quirk. 255 seems to be the exit code it uses but we don't know the reason, so we catch it and refer to the log -->
+    <Error Condition="'$(DocFXExitCode)' != '255' AND '$(DocFXExitCode)' != '0'" Text="DocFX build failed with exit code $(DocFXExitCode). Check docfx-log.txt for details." />
```

Comment:
in the Error Text it says `Check docfx-log.txt` but the log file is called `docfx.log`.